config.yml

```diff
@@ -0,0 +1,12 @@
+lang: en
```

Comment:
What do you need this config for? It has some strange components.

Reply:
That's my "fiddle-around" config file. It should be removed. There is a spaCy binding with Stanza so I originally figured I should explore that. Unfortunately it is implemented in a way that is currently incompatible with Rasa.